ext/standard/tests/array/array_key_exists_variation3.phpt

```diff
@@ -1,51 +0,0 @@
---TEST--
-Test array_key_exists() function : usage variations - floats and casting to ints
```

Comment:
I think this one is important to keep -- array_key_exists behavior with floats.

Reply:
Restored it